Lib/http/server.py

```diff
@@ -1051,9 +1051,14 @@ def finish_request(self, request, client_address):
             self.RequestHandlerClass(request, client_address, self,
                                      directory=args.directory)
 
+    class HTTPDualStackServer(DualStackServerMixin, ThreadingHTTPServer): pass
+    class HTTPSDualStackServer(DualStackServerMixin, ThreadingHTTPSServer): pass
```

Comment:
```suggestion
    class HTTPDualStackServer(DualStackServerMixin, ThreadingHTTPServer):
        pass
    class HTTPSDualStackServer(DualStackServerMixin, ThreadingHTTPSServer):
        pass
```

Sorry, my suggestion was just quick and dirty and not PEP-8 compliant.